### PR TITLE
Add manual test plan for the refactoring Set Tolerations For System Components and User-deployed Components Separately

### PR DESCRIPTION
**Pull request by PhanLe1010**
longhorn/longhorn#2120

### CODE REVIEW

File: docs/content/manual/release-specific/v1.1.1/tolerations_setting.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+## Related issue
+https://github.com/longhorn/longhorn/issues/2120
+
+Manual Tests:
+
+### Case 1: Existing Longhorn installation
+1. Install Longhorn master.
+1. Change toleration in UI setting
+1. Verify that `longhorn.io/last-applied-tolerations` annotation and `toleration` of manager, drive deployer, UI are not changed.
+1. Very that `longhorn.io/last-applied-tolerations` annotation and `toleration` for managed components (CSI components, IM pods, share manager pod, EI daemonset) are updated correctly
```
Comment (innobead) [inline docs/content/manual/release-specific/v1.1.1/tolerations_setting.md:15]: Probably we also need to consider `backing-image-manager` pods?

File: docs/content/manual/release-specific/v1.1.1/tolerations_setting.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
+title: Set Tolerations For System Components and User-deployed
```
Comment (innobead) [inline docs/content/manual/release-specific/v1.1.1/tolerations_setting.md:2]: sugget to change to `Set Tolerations/PriorityClass For System Components`

Q: what do you mean `user-deployed`?

**Review by innobead — CHANGES_REQUESTED** [review]
In general, LGTM

But, some comments needs to resolve/answer. Thanks!

File: docs/content/manual/release-specific/v1.1.1/tolerations_setting.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+### Case 2: New installation by Helm
+1. Install Longhorn master, set tolerations like:
+  ```
+  defaultSettings:
+    taintToleration: "key=value:NoSchedule"
+
+  longhornManager:
+    priorityClass: ~
+    tolerations:
+    # If you want to set tolerations for Longhorn Manager DaemonSet, delete the `[]` in the line above
```
Comment (khushboo-rancher) [inline docs/content/manual/release-specific/v1.1.1/tolerations_setting.md:26]: This inline comment is not required.

File: docs/content/manual/release-specific/v1.1.1/tolerations_setting.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    # and uncomment this example block
+    - key: key
+      operator: Equal
+      value: value
+      effect: NoSchedule   
+  ```
+3. Verify that the toleration is added for: IM pods, Share Manager pods, CSI deployments, CSI daemonset, the backup jobs, manager, drive deployer, UI
+4. Uninstall the Helm release. Verify that the uninstallation success
```
Comment (khushboo-rancher) [inline docs/content/manual/release-specific/v1.1.1/tolerations_setting.md:54]: If I have taints added for the node where I want to run the uninstaller job, the tolerations should be added to that uninstaller yaml file too, isn't it?

**Review by khushboo-rancher — COMMENTED** [review]
LGTM, left minor comments.

File: docs/content/manual/release-specific/v1.1.1/tolerations_setting.md (hunk 1 above)
Comment (PhanLe1010) [reply docs/content/manual/release-specific/v1.1.1/tolerations_setting.md]: Thank you. Fixed

File: docs/content/manual/release-specific/v1.1.1/tolerations_setting.md (hunk 2 above)
Comment (PhanLe1010) [reply docs/content/manual/release-specific/v1.1.1/tolerations_setting.md]: Thank you. Fixed
`user-deployed` components are components created directly by user (i.g., Longhorn manager, driver deployer, and UI) as oppose to to system components are the components created by Longhorn (CSI components, IM pods, share manager pod, EI daemonset, backing-image-manager, cronjob)

File: docs/content/manual/release-specific/v1.1.1/tolerations_setting.md (hunk 3 above)
Comment (PhanLe1010) [reply docs/content/manual/release-specific/v1.1.1/tolerations_setting.md]: Thank you. Fixed

File: docs/content/manual/release-specific/v1.1.1/tolerations_setting.md (hunk 4 above)
Comment (PhanLe1010) [reply docs/content/manual/release-specific/v1.1.1/tolerations_setting.md]: The uninstalling job will have the same toleration as Longhorn manager ds if users install by Helm and set toleration for manager ds.

**Review by khushboo-rancher — APPROVED** [review]

**Review by innobead — APPROVED** [review]

**Pull request merged**